packages/shared/src/styles/components/buttons.css

```diff
@@ -1,6 +1,6 @@
 .btn {
   transition: 0.2s ease-in-out;
-  transition-property: border, background, box-shadow;
+  transition-property: border, background, box-shadow, color;
```

Comment:
I noticed when hovering the items in the profile menu, the switch on the font colors is quite snappy so I applied the same transition to the color as well.